topostats/filters.py

```diff
@@ -202,12 +224,12 @@ def align_rows(self, image: np.ndarray, mask=None) -> np.ndarray:
 
         # Adjust the row medians accordingly
         # FIXME : I think this can be done using arrays directly, no need to loop.
-        for i in range(image.shape[0]):
+        for i in range(image_cp.shape[0]):
             # if np.isnan(row_median_diffs[i]):
             #     LOGGER.info(f"{i} Row_median is nan! : {row_median_diffs[i]}")
-            image[i] -= row_median_diffs[i]
+            image_cp[i] -= row_median_diffs[i]
```

Comment:
Surprised this doesn't fail, where have we defined `row_median_diffs` within this function? :thinking:

Reply:
It is defined 7 lines above:
`row_median_diffs = self._row_median_diffs(row_medians, median_row_height)`
Although the var `row_median_diffs` and the func `row_median_diffs` make things a little confusing. Should I rewrite these names?